### ISSUE TITLE
[bug]: Website broken: https://ui.shadcn.com/blocks

### ISSUE THREAD

**Issue by mariusk** [shadcn-ui/ui#5992]
### Describe the bug

Blocks part of the documentation website does not work (https://ui.shadcn.com/blocks).

All blocks are blanks, console log gets filled with "Minified React error ..." messages.

### Affected component/components

Website documentation

### How to reproduce

1. Go to https://ui.shadcn.com/blocks
2. Try to view any block

### System Info

```bash
Tested in Chrome (including incognito) and Safari
```

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues

**Comment by insivika** [comment]
This has been broken for weeks! @shadcn can we please get this looked at asap 🙏

**Comment by ev3nst** [comment]
it seems more like in the process of re-write not an error causing it to not render properly, this issue might take longer than people expect

**Comment by shadcn** [comment]
hmm I still cannot reproduce this? Is this on windows?

**Comment by GenhaoLi** [comment]
> All blocks are blanks

Do you mean these blocks?
![image](https://github.com/user-attachments/assets/4b8b7b5e-9fd9-463e-b744-f4850bd1d1e8)

I think that's what they are supposed to be. Check: in the code of the first example, lines 39-46:
```html
        <div className="flex flex-1 flex-col gap-4 p-4">
          <div className="grid auto-rows-min gap-4 md:grid-cols-3">
            <div className="aspect-video rounded-xl bg-muted/50" />
            <div className="aspect-video rounded-xl bg-muted/50" />
            <div className="aspect-video rounded-xl bg-muted/50" />
          </div>
          <div className="min-h-[100vh] flex-1 rounded-xl bg-muted/50 md:min-h-min" />
        </div>
```

The list of "blocks" in this page probably means "examples", if I understand correctly

**Comment by insivika** [comment]
Yes we exepect to see example blocks when selecting any of the navigation items on the left. For example "Authetication" would display an example of an Signup/Signin Form.

**Comment by ev3nst** [comment]
@GenhaoLi @shadcn it was i think more rich in content, right now its more like showcase of sidebar functionality.

for example; (sep 5)
https://web.archive.org/web/20240917123729/https://ui.shadcn.com/blocks

**Comment by ChuckJonas** [comment]
As someone new to the framework, this feels like the part of the documentation that I'm missing the most.   I tried the archieve.org link above, and I see exactly what I want, but the `code` view doesn't work :/

**Comment by insivika** [comment]
@shadcn can we have this back please

**Comment by shinusuresh** [comment]
Please bring this back

**Comment by shadcn** [comment]
This issue has been automatically marked as stale due to one year of inactivity. It will be closed in 7 days unless there’s further input. If you believe this issue is still relevant, please leave a comment or provide updated details. Thank you. (This is an automated message)

**Comment by shadcn** [comment]
This issue has been automatically closed due to one year of inactivity. If you’re still experiencing a similar problem or have additional details to share, please open a new issue following our current issue template. Your updated report helps us investigate and address concerns more efficiently. Thank you for your understanding! (This is an automated message)
